manifests/init.pp

```diff
@@ -1,41 +1,41 @@
 # == Class: crypto_win
 class crypto_win (
 
-  Boolean $event_logging  = true,
-  Boolean $multi_client   = false,
-  Boolean $multi_server   = false,
-  Boolean $tls_1_0_client = false,
-  Boolean $tls_1_0_server = false,
-  Boolean $tls_1_1_client = false,
-  Boolean $tls_1_1_server = false,
-  Boolean $tls_1_2_client = true,
-  Boolean $tls_1_2_server = true,
-  Boolean $ssl_2_0_client = false,
-  Boolean $ssl_2_0_server = false,
-  Boolean $ssl_3_0_client = false,
-  Boolean $ssl_3_0_server = false,
-  Boolean $pct_1_0_client = false,
-  Boolean $pct_1_0_server = false,
-  Boolean $md5            = false,
-  Boolean $sha            = false,
-  Boolean $sha256         = true,
-  Boolean $sha384         = true,
-  Boolean $sha512         = true,
-  Boolean $pkcs           = true,
-  Boolean $ecdh           = true,
-  Boolean $diffie_hellman = true,
-  Boolean $aes_128_128    = true,
-  Boolean $aes_256_256    = true,
-  Boolean $rc2_40_128     = false,
-  Boolean $rc2_56_128     = false,
-  Boolean $rc2_128_128    = false,
-  Boolean $rc4_40_128     = false,
-  Boolean $rc4_56_128     = false,
-  Boolean $rc4_64_128     = false,
-  Boolean $rc4_128_128    = false,
-  Boolean $des_56_56      = false,
-  Boolean $threedes_168   = false,
-  Boolean $null           = false,
+  Integer $event_logging_level = 3,
```

Comment:
https://puppet.com/docs/puppet/4.10/lang_data_number.html#the-integer-data-type

Suggestion:

```
Integer[0,7] $event_logging_level = 3,
```

.. this would have puppet check for acceptable range.

Reply:
implemented. thanks!!!